saleor/graphql/order/mutations/orders.py

```diff
@@ -781,6 +785,12 @@ def perform_mutation(cls, _root, info, **data):
         )
 
         recalculate_order(order)
+        if order.status == OrderStatus.DRAFT:
```

Comment:
This block of code is repeated a few times, maybe create function like `get_function_by_order_status`?

Reply:
fixed it. Is it acceptable now? It could be done all in one line in lambda but I though this way was more readable.